src/cozmo/pets.py

```diff
@@ -33,7 +33,7 @@
 '''
 
 # __all__ should order by constants, event classes, other classes, functions.
-__all__ = ['PET_VISIBILITY_TIMEOUT', 'PET_TYPE_UNKOWN', 'PET_TYPE_CAT', 'PET_TYPE_DOG',
+__all__ = ['PET_VISIBILITY_TIMEOUT', 'PET_TYPE_UNKNOWN', 'PET_TYPE_CAT', 'PET_TYPE_DOG',
```

Comment:
The constants should be in alpha order starting with 'PET_TYPE_CAT', 'PET_TYPE_DOG', etc.